### PR TITLE
Time and dimensions

### PR DESCRIPTION
**Pull request by mloy**
Still in work...

It would be nice to have some comments from others.

### CODE REVIEW

File: dimensions.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+- value dimension id: There is a fixed id for each value dimension of a signal.
+- name: Name of the dimension (i.e. voltage, sound level, time)
+- valueType: Describes the data type of the dimension (i.e. a number format ("u8", "u32", "s32", "u64", "s64", "real32", "real64"), time (We talked about this in prior sessions), other known types)
+- unit: Unit of the dimension (Out of scope of this document)
+- delta: (A value according to valueType) If this parameter does exist, the values of this dimension .
+  are equidistant. There will be no absolute value for this dimesnsion in the delivered data blocks. 
+  The absolute coordinate of the dimension has to be calculated using an absolute start value, 
+  the delta and the number of points delivered.
+  
+  If this parameter is missing for a dimension, each delivered point in measured value data blocks will carry a absolute value for the dimension.
```
Comment (gatzka) [inline dimensions.md]: Please put this it to the bullet point for "delta". At first I had no clue where this comment belongs to.

Comment (mloy) [reply dimensions.md]: true!

File: dimensions.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+      "endian": <string>,
+      "data": 
+      {
+    	"arraySize: <number>
+      },
+      "time":
+      {
+        "delta", <time object>
+        "time": <time object>
+        "size": <number>
```
[the remark this answers is not in the document]
Comment (mloy) [reply dimensions.md]: I removed the size

File: dimensions.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+
+~~~~ {.javascript}
+{
+  "method": "signal",
+  "params" : 
+    {
+      "endian": <string>,
+      "data": 
+      {
+    	"arraySize: <number>
```
[the remark this answers is not in the document]
Comment (mloy) [reply dimensions.md]: I added something about array size. I'm not sure about the name itself. Furthermore I'm not sure whether it is a series of points or a sequence of points I'm talking about...

**Pull request closed without merging**